lib/qxcompiler/resources/SCSSHandler.js

```diff
@@ -0,0 +1,100 @@
+/* ************************************************************************
+ *
+ *    qxcompiler - node.js based replacement for the Qooxdoo python
+ *    toolchain
+ *
+ *    https://github.com/qooxdoo/qxcompiler
+ *
+ *    Copyright:
+ *      2011-2017 Zenesis Limited, http://www.zenesis.com
+ *
+ *    License:
+ *      MIT: https://opensource.org/licenses/MIT
+ *
+ *      This software is provided under the same licensing terms as Qooxdoo,
+ *      please see the LICENSE file in the Qooxdoo project's top-level directory
+ *      for details.
+ *
+ *    Authors:
+ *      * John Spackman (john.spackman@zenesis.com, @johnspackman)
+ *
+ * ************************************************************************/
+
+var fs = require("fs");
+var path = require("path");
+var async = require('async');
+var sass = require('node-sass');
+var qx = require("qooxdoo");
+var util = require("../../util");
+require("./Handler");
+
+var log = util.createLog("resource-manager");
+
+qx.Class.define("qxcompiler.resources.SCSSHandler", {
```

Comment:
Qooxdoo coding style is to treat abbreviations as a capitalised word, eg `ScssHandler` instead of SCSSHandler